### DISCUSSION TITLE
Add timer during runtime

### DISCUSSION THREAD

**Discussion by jo47011** [mhthies/smarthomeconnect#87] [Ideas]
As of now the `_TimerSupervisor` only starts and monitors timers added before startup.

I want to create a timer when a window is opened.  Based on the current temperature I want an event to inform the user let's say every 20 minutes until the windows is closed again.  I have implemented it w/ my own AsnycTimer class but maybe it would make sense to enhance the class `_TimerSupervisor` to allow for adding `timer_tasks` after startup of the system so they get monitored as well.

**Comment by mhthies** [comment]
I'd like to keep the SHC setup as static as possible, i.e. create all objects before start up. This way, we don't have to worry about any kind of dynamic changes to connectable objects during operation.

My current idea for this problem is #45: You would statically create a standard [`shc.timer.Every`](https://smarthomeconnect.readthedocs.io/en/stable/timer.html#shc.timer.Every) timer with an interval of 20 minutes and without alignment , but only enable it as long as the window is open.
